Experiment_Launcher_code/ExperimentLauncher.py

```diff
@@ -11,6 +11,7 @@
 
 if not __DISABLE_DATA_ANALYZER:
     from Data_analysis.DataAnalysisScript import DataAnalyzer
+    from Data_analysis.event_logger import EventLogger
```

Comment:
Event logger is created in the experiment manager - I think we do not need it here